browser/android/preferences/brave_pref_service_bridge.cc

```diff
@@ -45,9 +44,6 @@ Profile* GetOriginalProfile() {
 }
 
 const char* GetPrefNameExposedToJava(int pref_index) {
-  if (pref_index < kBravePrefOffset)
```

Comment:
I'm confused, if `PrefServiceBridge::GetPrefNameExposedToJava` is not called then this method doesn't make sense anymore

Reply:
it seems like we're missing a bunch of other changes that are necessary to bring our android prefs inline with upstream. For instance they have removed chrome/browser/android/preferences/prefs.h and we should also change our usage of `GetPrefNameExposedToJava` to match `base::android::ConvertJavaStringToUTF8`

Reply:
@samartnik let's do a follow-up for this to mostly get rid of `BravePrefServiceBridge` by adding our prefs to `java_pref_names_srcjar` in chrome/browser/preferences/BUILD.gn

Reply:
https://github.com/brave/brave-browser/issues/11122